sdk/communication/Azure.Communication.Common/tests/Pipeline/HMACAuthenticationPolicyTests.cs

```diff
@@ -22,13 +22,13 @@ public async Task TestHMACPolicyProcess()
             await SendGetRequest(transport, authPolicy);
             var headers = transport.SingleRequest.Headers;
 
-            Assert.True(headers.Contains("Date"));
```

Comment:
nit: Date still exists right? Can we just add a new assert.true for `x-ms-date` instead instead of removing the check for `Date`?